docs/core_modules/data_modules/storage/index_stores.md

```diff
@@ -39,7 +39,7 @@ since data is persisted by default.
 
 You can easily reconnect to your MongoDB collection and reload the index by re-initializing a `MongoIndexStore` with an existing `db_name` and `collection_name`.
 
-A more complete example can be found [here](../../examples/docstore/MongoDocstoreDemo.ipynb)
+A more complete example can be found [here](https://github.com/ravi03071991/gpt_index/blob/main/docs/examples/docstore/MongoDocstoreDemo.ipynb)
```

Comment:
Actually, we should copy these remaining doctor examples into `llama_index/docs/examples/docstore`

Reply:
Sure. Will do that.